### ISSUE TITLE
Updatable HelpContent zip file

### ISSUE THREAD

**Issue by ThomasNieto** [MicrosoftDocs/PowerShell-Docs#9376]
### Prerequisites

- [X] **Existing Issue:** Search the existing issues for this repository. If there is an issue that fits your needs do not file a new one. Subscribe, react, or comment on that issue instead.
- [X] **Descriptive Title:** Write the title for this issue as a short synopsis. If possible, provide context. For example, "Typo in `Get-Foo` cmdlet" instead of "Typo."
- [X] **Verify Version:** If there is a mismatch between documentation and the behavior on your system, ensure that the version you are using is the same as the documentation. Check this box if they match or the issue you are reporting is not version specific.

### Links

https://github.com/MicrosoftDocs/PowerShell-Docs/blob/main/reference/docs-conceptual/developer/help/updatable-help-authoring-step-by-step.md

### Summary

It looks like `Update-Help` uses the cab file on Windows and zip file on Unix according to the code. The updatable help step-by-step guide only refers to creating the cab file. If zip file used on Unix then the guide should be updated to reflect that.

https://github.com/PowerShell/PowerShell/blob/dd884459c1ccafc3fe7615d53a0ffa7b9646c571/src/System.Management.Automation/help/UpdatableHelpModuleInfo.cs#L14-L17 

### Suggested Fix

If zip help content file is used then mention it on the step-by-step guide.

**Comment by sdwheeler** [comment]
These are the legacy Windows PowerShell SDK docs. PlatyPS is the tool to create updateable help. It creates both CAB and ZIP files for you. See https://learn.microsoft.com/powershell/utility-modules/platyps/create-help-using-platyps

**Comment by ThomasNieto** [comment]
I did not notice it was under the legacy SDK section until you pointed it out. Its very difficult to see that these docs are legacy unless you scroll up in the navigation tree until you see that section header. Why the legacy SDK included in the PowerShell 7 drop down and not separated into another section to prevent confusion?

If these pages cannot be separated it would be nice if it had a call-out saying use this page instead and this is legacy info.

Also, that page you link to also references back to this page for more information.

> For more information, see [Supporting Updateable Help: Step-by-step](https://learn.microsoft.com/en-us/powershell/scripting/developer/help/updatable-help-authoring-step-by-step).

While it does say to copy both the cab and zip to the web server I feel this statement is misleading. It doesn't mention the information I found in the code that Windows uses the cab file while Unix uses the ZIP.

> CAB files are compatible with all versions of PowerShell. PowerShell 6 and higher can use ZIP files.
